backend/src/services/v2/smtp/smtp.ts

```diff
@@ -86,6 +88,80 @@ export async function requestReviewForAccessRequest(
   await Promise.all(sendEmailResponses)
 }
 
+export async function notifyReviewResponseForRelease(review: ReviewDoc, release: ReleaseDoc) {
+  if (!config.smtp.enabled) {
+    log.info('Not sending email due to SMTP disabled')
+    return
+  }
+  const reviewResponse = review.responses[0]
+
+  if (!reviewResponse) {
+    log.info('response not found')
+    return
+  }
+  const emailContent = buildEmail(
+    `Release ${release.semver} has been reviewed by ${reviewResponse?.user}`,
+    [
+      { title: 'Model ID', data: release.modelId },
+      { title: 'Decision', data: reviewResponse.decision.replace(/_/g, ' ') },
```

Comment:
Maybe for both access requests and releases, you could have 'Reviewer Role' instead of calling what you had before 'Your Role'?